app/__init__.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import logging
+
+app = Flask(__name__)
+
+# Initialize MariaDB connection
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://root:@localhost:3306/inventory'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['LOGGING_LEVEL'] = logging.INFO
+
+db = SQLAlchemy(app)
+
+from app import server, models
```

Comment:
what is this line aiming for?

Reply:
I believe line 14 is for the convenience of importing module, i.e. one can import the modules from outside the package using `from app import server` instead of `from app.server import server`